Scripts/Editor/AssetBundleWindow.cs

```diff
@@ -112,6 +119,107 @@ void OnGUI()
+                                            RenderTextureFormat.Default,
+                                            RenderTextureReadWrite.Default);
+                        // Blit the pixels on texture to the RenderTexture
+                        Graphics.Blit(prefabPreview, tmp);
+                        // Backup the currently set RenderTexture
+                        RenderTexture previous = RenderTexture.active;
+                        // Set the current RenderTexture to the temporary one we created
+                        RenderTexture.active = tmp;
+                        // Create a new readable Texture2D to copy the pixels to it
+                        Texture2D myTexture2D = new Texture2D(prefabPreview.width, prefabPreview.height);
+                        // Copy the pixels from the RenderTexture to the new Texture
+                        myTexture2D.ReadPixels(new Rect(0, 0, tmp.width, tmp.height), 0, 0);
+                        myTexture2D.Apply();
+                        // Reset the active RenderTexture
+                        RenderTexture.active = previous;
+                        // Release the temporary RenderTexture
+                        RenderTexture.ReleaseTemporary(tmp);
+                        byte[] bytes = myTexture2D.EncodeToPNG();
+                        imgPreviews[nameWOext] = bytes;
+                        imgPreviews[nameWOext.ToLower()] = bytes;
+
+
+
+                        if (!Directory.GetDirectories(Application.dataPath + assetBundleFolderLocation).Contains(nameWOext))
+                        {
+                            Directory.CreateDirectory(Application.dataPath + assetBundleFolderLocation + "/" + nameWOext);
+                            //File.Create(Application.dataPath + assetBundleFolderLocation + "/" + nameWOext + ".meta");
+                        }
+                        // move file
+                        File.Copy(asset, Application.dataPath + assetBundleFolderLocation + "/" + nameWOext + "/" + "model" + ".prefab", true);
+
+                    }
+                    /*else if (asset.EndsWith(".prefab.meta"))
+                    {
+                        Regex r = new Regex(@".+[/\\]+");
+                        string nameWOext = r.Replace(asset.Replace(".prefab.meta", ""), ""); //remove extension and all previous directories
+                        if (!Directory.GetDirectories(Application.dataPath + assetBundleFolderLocation).Contains(nameWOext))
+                        {
+                            Directory.CreateDirectory(Application.dataPath + assetBundleFolderLocation + "/" + nameWOext);
+                        }
+                        // move file
+                        File.Copy(asset, Application.dataPath + assetBundleFolderLocation + "/" + nameWOext + "/" + "model" + ".prefab.meta", true);
+
+                    }*/
+                }
+
+                //reload asset directory
+                AssetDatabase.Refresh();
```

Comment:
ignores metadata moving/copying and instead generates new metadata for the newly created folders and files